src/swash.rs

```diff
@@ -34,7 +34,14 @@ fn swash_image(
 
     // Compute the fractional offset-- you'll likely want to quantize this
     // in a real renderer
-    let offset = Vector::new(cache_key.x_bin.as_float(), cache_key.y_bin.as_float());
+    let offset = if cache_key.flags.contains(CacheKeyFlags::PIXEL_FONT) {
+        Vector::new(
+            cache_key.x_bin.as_float().round() + 1.0,
```

Comment:
Why is 1.0 added here?

Reply:
I have no idea, but without it the font is shifted 1px to the left compared to the Freetype render I've been using as a comparison.

Reply:
This is acceptable for now. I think in the future we will have a better way to deal with this - but it will require hinting changes at a lower level.